packages/appkit-utils/src/CaipNetworkUtil.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  ConstantsUtil,
+  type AppKitNetwork,
+  type CaipNetwork,
+  type CaipNetworkId
+} from '@reown/appkit-common'
+import { PresetsUtil } from './PresetsUtil.js'
+
+const RPC_URL_HOST = 'rpc.walletconnect.org'
+
+export function getBlockchainApiRpcUrl(caipNetworkId: CaipNetworkId, projectId: string) {
+  return `https://rpc.walletconnect.org/v1/?chainId=${caipNetworkId}&projectId=${projectId}`
+}
+
+const WC_HTTP_RPC_SUPPORTED_CHAINS = [
```

Comment:
Maybe we could move to another place?

Reply:
do we want to keep and mantain this here? shouldn't we query this from an api call?

Reply:
I thought about it but I don't want to make the initialization slower. It'll add extra 200-300 ms time (maybe more depending on the internet)